webpack/sass/main.scss

```diff
@@ -207,6 +207,11 @@ code {
     border-bottom: 0.1875rem solid $color-light-gray;
   }
 
+  .column:nth-last-child(-n + 3):nth-child(3n + 1), .column:nth-last-child(-n + 3):nth-child(3n + 1) ~ .column
```

Comment:
Please add a css comment explaining how this selector works for other users in the future, but it works great!